### ISSUE TITLE
Removing entities from device causes error

### ISSUE THREAD

**Issue by OldShaterhan** [rospogrigio/localtuya#937]
## The problem

When I try to unswitch one of the entities (using edit device) I get an error and in logs:

## Environment

- Localtuya version:  4.0.1
- Last working localtuya version (if known and relevant): 
- Home Assistant Core version:  2022.6.7- [x] Are you using the Home Assistant Tuya Cloud component ? 
- [] Are you using the Tuya App in parallel ?  (only for adding items - SmartLife)

## Steps to reproduce

1. Chose edit device option
2. Uncheck one of the entities.
3. An error ocucrs.

## Configuration `configuration.yaml` or `config_flow`

```yaml

```

## Provide Home Assistant taceback/logs

```txt
Traceback (most recent call last):
  File "/config/custom_components/localtuya/config_flow.py", line 533, in async_step_configure_device
    self.dps_strings = await validate_input(self.hass, user_input)
  File "/config/custom_components/localtuya/config_flow.py", line 237, in validate_input
    data[CONF_DEVICE_ID],
KeyError: 'device_id'
```

**Comment by jf647** [comment]
I am also seeing this error, both on 2022.6.7 and on the just-released 2022.7.0

```
Logger: custom_components.localtuya.config_flow
Source: custom_components/localtuya/config_flow.py:237
Integration: LocalTuya ([documentation](https://github.com/rospogrigio/localtuya/), [issues](https://github.com/rospogrigio/localtuya/issues))
First occurred: 7:15:44 PM (1 occurrences)
Last logged: 7:15:44 PM

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/localtuya/config_flow.py", line 533, in async_step_configure_device
    self.dps_strings = await validate_input(self.hass, user_input)
  File "/config/custom_components/localtuya/config_flow.py", line 237, in validate_input
    data[CONF_DEVICE_ID],
KeyError: 'device_id'
```

I deleted the device (an RGB bulb) from the Tuya Smartlife app before I attempted to remove the LocalTuya device, if that makes any difference.

**Comment by FritsOV** [comment]
i also have the same error in latest localtuya.

Also deleting entitys is no problem but deleting the last left entity gives an error.

**Comment by nkgiovannivl** [comment]
I can delete as described here https://github.com/rospogrigio/localtuya/issues/871#issuecomment-1156923434

**Comment by CloCkWeRX** [comment]
Duplicate of #897 (though this is a better writeup)

**Comment by CloCkWeRX** [comment]
Able to reproduce

**Comment by LordShelmchen** [comment]
Same issue, able to reproduce
